README.md

```diff
@@ -41,6 +41,9 @@ All these variables are initialized with default variables defined in :code:`.en
 - `REMOTE_STORAGE`: defines if remote storage is used. If this variable is defined, then the LOVE-manager will connect to the LFA to upload files. If not defined, then the LOVE-manager will store the files locally.
 - `COMMANDING_PERMISSION_TYPE`: defines the type of permission to use for commanding. Currently two options are available: `user` and `location`. If `user` is used, then requests from users with `api.command.execute_command` permission are allowed. If `location` is used, then only requests from the configured location of control will be allowed. If not defined, then `user` will be used.
 - `URL_SUBPATH`: defines the path where the LOVE-manager will be served. If not defined, then requests will be served from the root path `/`. Note: the application has its own routing system, so this variable must be thought of as a prefix to the application's routes.
+- `SMTP_USER`: defines the user to use to send emails. The `@mail.lsst.org` domain is added automatically.
```

Comment:
Are you sure that the domain is `mail.lsst.org`? My email address has the `lsst.org` domain. Please review and correct if necessary.

Reply:
good catch! thank you :pray: